src/sql/workbench/services/urlBrowser/browser/urlBrowserDialogService.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { UrlBrowserDialog } from 'sql/workbench/services/urlBrowser/browser/urlBrowserDialog';
+import { IUrlBrowserDialogService } from 'sql/workbench/services/urlBrowser/common/urlBrowserDialogService';
+import { localize } from 'vs/nls';
+import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
+
+/**
+ * Url browser dialog service
+ */
+export class UrlBrowserDialogService implements IUrlBrowserDialogService {
```

Comment:
Same here about the name, we should make it clear this is for a specific purpose and not just general URLs